### ISSUE TITLE
deen Crashes on X509Certificate Formatting

### ISSUE THREAD

**Issue by SvenTo** [takeshixx/deen#18]
Steps to reproduce:
- Input: `MIIGITCCBQmgAwIBAgISA+Ru`
- Select Base64 decode
- Select in lower pane: Miscellaneous -> X509Certificate

![screenshot_20170328_160156](https://cloud.githubusercontent.com/assets/2223926/24409229/e5ed4c64-13cf-11e7-985d-679903d39d34.png)

Stack Trace:
````
$ deen
  OpenType support missing for script 9
[x509.py:27 -          certificate() ] Invalid certificate encoding
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/deen/widgets/encoder.py", line 326, in <lambda>
    self.misc_combo.currentIndexChanged.connect(lambda: self.action(self.misc_combo))
  File "/usr/lib/python3.6/site-packages/deen/widgets/encoder.py", line 509, in action
    self.next.content = transformer.decode()
  File "/usr/lib/python3.6/site-packages/deen/widgets/encoder.py", line 82, in content
    assert isinstance(data, (bytearray, bytes))
AssertionError
[1]    18240 abort (core dumped)  deen
````

**Comment by takeshixx** [comment]
Should be fixed in 7994789f6f9d1c90cc037805b2359c824e306ab7.

**Comment by takeshixx** [comment]
X509 formatting should now also support DER-encoded (non-Base64-encoded) certificates (299b3ad73d4e809f071ecdb3e306b13b3a04c75d).
